validator/accounts/v2/wallet_create.go

```diff
@@ -192,14 +192,17 @@ func inputKeymanagerKind(cliCtx *cli.Context) (v2keymanager.Kind, error) {
 	promptSelect := promptui.Select{
 		Label: "Select a type of wallet",
 		Items: []string{
-			wallet.KeymanagerKindSelections[v2keymanager.Derived],
 			wallet.KeymanagerKindSelections[v2keymanager.Direct],
+			wallet.KeymanagerKindSelections[v2keymanager.Derived],
 			wallet.KeymanagerKindSelections[v2keymanager.Remote],
 		},
 	}
 	selection, _, err := promptSelect.Run()
 	if err != nil {
 		return v2keymanager.Direct, fmt.Errorf("could not select wallet type: %v", prompt.FormatPromptError(err))
 	}
-	return v2keymanager.Kind(selection), nil
+	if selection == 2 {
+		return v2keymanager.Remote, nil
+	}
+	return v2keymanager.Kind(selection ^ 1), nil
```

Comment:
What's up with these changes?

Reply:
It's just a matter of taste to have the recommended wallet before the non-recommended ones. If you keep it as it was it would cast the wrong `v2keymanager.Kind`

Reply:
I guess is fine to have the original order and also remove these changes here. There's little to lose and we gain that this horrible shuffling of v2keymanager.Kind may break in the future some test or actual code.

Reply:
Oh, I see. What about reordering the enum to coincide with the order here?

Reply:
I don't know all your code and was afraid that could brake not only tests here but in other places. Since this code has low impact I figured it was safer than changing the enum.

Reply:
@rauljordan what do you think?

Reply:
I think we should leave such hacks for the mainnet and just reorder the enum. We should be able to quickly patch any introduced bugs. And we should have tests for such breaking changes anyway.

One possible strategy would be to see which tests fail after reordering the enum. If something does not fail although it should, then we ought to correct/add a test before making changes to the code that reflect the reordering. If something fails after the reordering, then this piece of code was covered by tests correctly.

Reply:
I changed the enum, will see here what tests (if any) are failing.